Fix Dockerfile

fixdocker opened the pull request:
Hello, I represent a research group investigating build failures in
Dockerfiles. We executed a Dockerfile in your project and found that
it is not building. We are sending this Pull Request with a suggested
repair. The repair operation is based on the following URL:

- https://github.com/docker-library/php/issues/662#issuecomment-397051998

Please note that we are using an anonymous GitHub account because of
double-blind restrictions in the submission process of conferences.

fixdocker committed:
Update Dockerfile

fixdocker commented:
Hey @JensErat @rwky , can you check this PR?

rwky commented:
I've not used selfoss in some time but looking at the repo it'd be better to upgraded to a supported PHP version (ideally 7.4)

fixdocker commented:
@rwky, we have changed the base image to `php: 7.4-apache`, and after that we also noticed problems with the `docker-php-ext` configuration so we changed from `--with-jpeg-dir=` to `--with-jpeg=`[1] and remove `mbstring`[2] , and this fixed it. Can we submit a new pull request with these changes?

[1] - https://github.com/docker-library/php/issues/931#issuecomment-568658449
[2] - https://stackoverflow.com/a/59253249
